packages/synapse-react-client/src/components/EntityFinder/details/configurations/ReferenceDetails.tsx

```diff
@@ -0,0 +1,37 @@
+import { Reference } from '@sage-bionetworks/synapse-types'
+import React from 'react'
+import { useGetEntityHeaders } from '../../../../synapse-queries/index'
+import { getIsAllSelectedFromInfiniteList } from '../../../../utils/hooks/index'
+import { EntityFinderHeader } from '../../EntityFinderHeader'
+import { EntityDetailsListSharedProps } from '../EntityDetailsList'
+import { DetailsView } from '../view/DetailsView'
+
+type SearchDetailsProps = EntityDetailsListSharedProps & {
+  references: Reference[]
+}
+
+export const ReferenceDetails: React.FunctionComponent<SearchDetailsProps> = ({
```

Comment:
The selected items will be provided as `Reference`s. This component translates the reference list to entity headers, which can be passed into `DetailsView` to render the table.